src/frontend/komponenter/Felleskomponenter/Datovelger/Datovelger.tsx

```diff
@@ -20,8 +20,15 @@ interface IProps {
     kanKunVelgeFremtid?: boolean;
     datoMåFyllesUt?: boolean;
     readOnly?: boolean;
+    disableWeekends?: boolean;
 }
 
+enum DatovelgerFeilmelding {
+    HELG_ER_UGYLDIG = 'HELG_ER_UGYLDIG',
+}
+
+type Feilmelding = GenerellFeilmelding | DatovelgerFeilmelding;
```

Comment:
Forslag på måte å løse det på. Vi har til nå bare hatt behov for felles feilmeldinger, men i denne PRen dukket det opp en som er spesifikk for Datovelgeren (Helg er ugyldig). Si gjerne i fra hvis dere ikke liker måten jeg har løst det på her.